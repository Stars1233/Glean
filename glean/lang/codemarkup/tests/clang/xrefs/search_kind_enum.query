query: search.kind.cxx.SearchEnum { name = _ }
perf: true
transform: [gensort, []]
