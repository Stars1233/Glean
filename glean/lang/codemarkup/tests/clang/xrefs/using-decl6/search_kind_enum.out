[
  "@generated",
  {
    "key": {
      "name": "Foo",
      "scope": { "namespace_": { "key": { "name": { "key": "N1" } } } },
      "entity": {
        "decl": {
          "enum_": {
            "key": {
              "name": {
                "key": {
                  "name": { "key": "Foo" },
                  "scope": {
                    "namespace_": { "key": { "name": { "key": "N1" } } }
                  }
                }
              },
              "isScoped": false,
              "source": {
                "file": { "key": "test.cpp" },
                "lineBegin": 10,
                "columnBegin": 1,
                "lineEnd": 10,
                "columnEnd": 12
              }
            }
          }
        }
      }
    }
  }
]